website/package.json

```diff
@@ -31,12 +31,13 @@
     "d3-scale": "^3.2.1",
     "expr-eval": "^2.0.2",
     "mapbox-gl": "^2.0.0",
```

Comment:
How hard would it be to drop mapbox completely here? Are there a few more examples in the repo that need to be updated?

Reply:
Not hard, I'll look into it.

Reply:
Fixed in #86